app/models/manageiq/providers/cisco_intersight/inventory/collector/physical_infra_manager.rb

```diff
@@ -48,50 +48,56 @@ def get_storage_api
       IntersightClient::StorageApi.new
     end
 
+    def get_network_api
```

Comment:
I think this would be better as just `network_api`, also since this really only needs to be initialized once you can memoize it for the duration of this refresh run like:
```
def network_api
  @network_api ||= IntersightClient::NetworkApi.new
end
```

Reply:
Ok, going to do that for all functions in collector that start provide same functionality (that is, for all code included in following lines: https://github.com/ManageIQ/manageiq-providers-cisco_intersight/blob/739331981c509ee52ddc1302fa08760dbbf64863/app/models/manageiq/providers/cisco_intersight/inventory/collector/physical_infra_manager.rb#L19-L49)